deploy/helm/templates/addons/mysql-addon.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: extensions.kubeblocks.io/v1alpha1
+kind: Addon
+metadata:
+  name: mysql
+  labels:
+    {{- include "kubeblocks.labels" . | nindent 4 }}
+    "kubeblocks.io/provider": community
+  {{- if .Values.keepAddons }}
+  annotations:
+    helm.sh/resource-policy: keep
+  {{- end }}
+spec:
+  description: 'MySQL is a widely used, open-source relational database management system (RDBMS).'
```

Comment:
Do link with feature issue, are we intend to add a new CD to distinguish ApeCloud MySQL vs Community MySQL just for replication support feature?

Reply:
@ruijun2002 do review this PR

Reply:
Yes, this will serve as a temporary version intended for testing purposes. The final version will be implemented by the RSM.